src/Servers/Kestrel/Core/test/Http1ConnectionTests.cs

```diff
@@ -728,25 +724,6 @@ public async Task RequestAbortedTokenIsResetBeforeLastWriteWithChunkedEncoding()
             Assert.False(_http1Connection.RequestAborted.IsCancellationRequested);
         }
 
-        [Fact]
-        public void RequestAbortedTokenIsFullyUsableAfterCancellation()
```

Comment:
Do we want to keep this test?

Reply:
One of the WaitHandle assertions fail now, but I removed it with a comment and re-added the test.